src/services/todo.services/todoServices.ts

```diff
@@ -18,6 +18,7 @@ export async function createToDo(
     description,
     id,
     createdDate,
+    userId: "random@email.com",
```

Comment:
If this was hardcoded just for testing purposes, we should delete it, but instead read the email from currently logged in user we get from session.